src/Tools/dotnet-user-jwts/test/UserJwtsTests.cs

```diff
@@ -480,4 +480,88 @@ public void Key_CanPrintAndReset_BySchemeAndIssuer()
         var resetKey = resetMatches.SingleOrDefault().Groups[1].Value;
         Assert.NotEqual(key, resetKey);
     }
+
+    [Fact]
+    public void Create_CanHandleNoProjectOptionProvided()
+    {
+        var projectPath = _fixture.CreateProject();
+
+        Directory.SetCurrentDirectory(Path.Combine(projectPath, ".."));
+
+        var app = new Program(_console);
+        app.Run(new[] { "create" });
+
+        Assert.DoesNotContain("No project found at `-p|--project` path, current directory, or child directories.", _console.GetOutput());
+        Assert.Contains("New JWT saved", _console.GetOutput());
+    }
+
+    [Fact]
+    public void Create_CanHandleNoProjectOptionProvided_WithNoProjects()
+    {
+        var projectPath = _fixture.CreateProject();
+        var projectParent = Path.Combine(projectPath, "..");
+
+        Directory.Delete(projectParent, true);
+        Directory.CreateDirectory(projectParent);
```

Comment:
This is an expensive way to create an empty folder

Reply:
Fair point!